vcd/resource_vcd_vapp_vm_capabilities_test.go

```diff
@@ -0,0 +1,110 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func init() {
+	testingTags["vm"] = "resource_vcd_vapp_vm_capabilities_test.go"
+}
+
+func TestAccVcdVAppVmCapabilities(t *testing.T) {
+	var vapp govcd.VApp
+	var vm govcd.VM
+
+	var params = StringMap{
+		"Org":         testConfig.VCD.Org,
+		"Vdc":         testConfig.VCD.Vdc,
+		"Catalog":     testSuiteCatalogName,
+		"CatalogItem": testSuiteCatalogOVAItem,
+		"VappName":    vappName2,
+		"VmName":      vmName,
+	}
+
+	configText := templateFill(testAccCheckVcdVAppVm_capabilities, params)
+
+	params["FuncName"] = t.Name() + "-step1"
+	configText1 := templateFill(testAccCheckVcdVAppVm_capabilitiesUpdate, params)
+
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+	debugPrintf("#[DEBUG] CONFIGURATION: %s\n", configText)
```

Comment:
Please add a print for configText1 as well

Reply:
added